### DISCUSSION TITLE
courses/beginners-web-handbook/javascript-quiz/01-basic

### DISCUSSION THREAD

**Discussion by giscus** [crwdu/lms-contents-js101#56] [Q&A] [answered]
# courses/beginners-web-handbook/javascript-quiz/01-basic

바닐라코딩

https://www.vcdr.club/courses/beginners-web-handbook/javascript-quiz/01-basic

**Comment by suyoung247** [comment, marked as answer]
4 - 3 퀴즈에서 문자열에서 숫자를 뺄 수 없어서 TypeError가 나와야 되는 것으로 알고 있는데 왜 값이 4가 나왔는지 궁금합니다.

**Reply by Ken123777** [reply]
더하기일 경우와 빼기일 경우에 각각의 값이 어떤 규칙에 의해 형변환 되는지 살펴보시면 도움이 될 것 같습니다. [링크](https://stackoverflow.com/a/24383830) 하나 첨부드리니, 살펴보시고 이해가 안되시면 또 말씀해주세요! 😃
